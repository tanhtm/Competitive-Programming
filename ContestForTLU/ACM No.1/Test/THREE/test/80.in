841719709000000000
