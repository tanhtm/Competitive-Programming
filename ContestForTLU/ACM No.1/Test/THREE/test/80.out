12 16 17 26 27 28 30 34 35 37 
0 3 4 5 6 7 9 10 11 13 20 21 22 24 25 29 31 32 33 38 
